src/pages/signup/SignUpPage.module.css

```
@@ -0,0 +1,129 @@
+.logo {
+  width: 3rem;
+  height: 3rem;
+  color: #4f46e5;
+}
+
+.title {
+  font-size: 1.875rem;
+  font-weight: bold;
+  color: #111827;
+  margin-left: 0.5rem;
+  margin: 0;
```

Biome (2.1.2)

[error] Unexpected shorthand property margin after margin-left



(lint/suspicious/noShorthandPropertyOverrides)